### PR TITLE
bugfix: fix prediction service and unit tests

### PR DESCRIPTION
**Pull request by dizcology**
Related issues:
https://github.com/googleapis/gapic-generator-python/issues/413
https://github.com/googleapis/gapic-generator-python/issues/414

### CODE REVIEW

File: google/cloud/aiplatform_v1beta1/services/prediction_service/client.py [hunk 1]
```diff
@@ -328,7 +328,7 @@ def explain(
         if endpoint is not None:
             request.endpoint = endpoint
         if instances is not None:
-            request.instances = instances
```
Comment (busunkim96) [inline google/cloud/aiplatform_v1beta1/services/prediction_service/client.py:331]: Synth will try to revert changes like this in future Autosynth PRs. You can add this entire file to the `exclude` list, or a write a regex replacement in `synth.py`.

**Review by busunkim96 — APPROVED** [review]

File: google/cloud/aiplatform_v1beta1/services/prediction_service/client.py (hunk 1 above)
Comment (dizcology) [reply google/cloud/aiplatform_v1beta1/services/prediction_service/client.py:331]: I had this in `synth.py` but failed to update the relevant line to v1beta1, it has been fixed: https://github.com/googleapis/python-aiplatform/pull/8/files#diff-3620890cb174a7f98c0722895e37bb79R72

Comment (software-dov) [reply google/cloud/aiplatform_v1beta1/services/prediction_service/client.py:331]: I can also reprioritize the issues that cause the need for synth hacks. The repeated field one should be trivial.

Comment (software-dov) [reply google/cloud/aiplatform_v1beta1/services/prediction_service/client.py:331]: Oh, my bad, it looks like that issue has actually been fixed in a more current generator release.

**Pull request merged**